### PR TITLE
Docs: updated configuration description for auto_assign_org

### PR DESCRIPTION
**Pull request by gjed**
**What this PR does / why we need it**:
Added additional description to clarify what will happen when, in the grafana configuration, both `allow_org_create` and `auto_assign_org` are set to `false`.

**Which issue(s) this PR fixes**: 
none

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by gjed** [comment]
Sorry, I made a bit of a mess with my repo.. I fixed it but it still pulled in quite a few people as reviewers.
Also, I signed the CLA, but the CLAssistent doen't seem to pick it up

**Review by brendamuir — APPROVED** [review]

File: docs/sources/setup-grafana/configure-grafana/_index.md [hunk 1]
```diff
@@ -675,7 +675,8 @@ Default is `false`.
 
 Set to `true` to automatically add new users to the main organization
 (id 1). When set to `false`, new users automatically cause a new
-organization to be created for that new user. Default is `true`.
+organization to be created for that new user. The organization will be 
+created even if the above setting is set to `false`. Default is `true`.
```
[the remark this answers is not in the document]
Comment (gjed) [reply docs/sources/setup-grafana/configure-grafana/_index.md:679]: Yes, `the above setting` is referring to `allow_org_create`. 
You're right, that is much more clear.

**Comment by gjed** [comment]
Please review the issue

File: docs/sources/setup-grafana/configure-grafana/_index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (gguillotte-grafana) [reply docs/sources/setup-grafana/configure-grafana/_index.md:679]:
```suggestion
organization to be created for that new user. The organization will be 
created even if the `allow_org_create` setting is set to `false`. Default is `true`. 
```

**Review by gguillotte-grafana — APPROVED** [review]

**Comment by gguillotte-grafana** [comment]
Failing on linting, probably on something unrelated.

File: docs/sources/setup-grafana/configure-grafana/_index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (osg-grafana) [reply docs/sources/setup-grafana/configure-grafana/_index.md]: This had a trailing space on that suggested line.

File: docs/sources/setup-grafana/configure-grafana/_index.md [hunk 2]
```diff
@@ -675,7 +675,9 @@ Default is `false`.
 
 Set to `true` to automatically add new users to the main organization
 (id 1). When set to `false`, new users automatically cause a new
-organization to be created for that new user. Default is `true`.
+organization to be created for that new user. The organization will be 
+organization to be created for that new user. The organization will be
```
Comment (osg-grafana) [inline docs/sources/setup-grafana/configure-grafana/_index.md:679]:
```suggestion
organization to be created for that new user. The organization will be
```

**Comment by osg-grafana** [comment]
Removed trailing space to see if this rebuilds properly.

**Comment by gguillotte-grafana** [comment]
@osg-grafana Please see https://github.com/gjed/grafana/pull/1 which applies the changes from `prettier --write`. I am starting to suspect applying suggestions in GitHub commits introduces issues with `prettier`. (Also, the applied suggestion added a duplicate line.)

**Comment by gjed** [comment]
Is this ok?
Logs says:
```
 This is expected for PRs coming from forks, please check that the changes are compatible with Enterprise before merging this PR.
```

**Comment by gguillotte-grafana** [comment]
That check isn't necessarily relevant for docs PRs, so I've merged this PR. Thanks!

**Pull request merged**